### ISSUE TITLE
Getting "slot" under the slot tab

### ISSUE THREAD

**Issue by danielgroen** [storybookjs/storybook#14378]
Hi, 

i'm wondering how i can get the property under "SLOTS"

See below the appearence:
![afbeelding](https://user-images.githubusercontent.com/6578052/112852598-5e8cea00-90ac-11eb-9e95-7ce55a7a3b1c.png)

The code i use is:
```js
const Template = (args, { argTypes }) => ({
  components: { AtomTitle },
  props: Object.keys(argTypes),
  template: '<AtomTitle v-bind="$props">{{ slot }}</AtomTitle>',
})

// TODO:: Slot must be in storybook controls slot section. Now it got it's own section
export const Title = Template.bind({})
Title.args = {
  type: 'h2',
  color: 'black',
  slot: 'Title',
}
```

Any idea? I cannot find anything about this in the docs under "controls"

**Comment by Cubid** [comment]
Hello,

There is an example here: https://highlandsolutions.com/blog/stop-designing-components-in-your-vue-application

But I am unable to get it working for the moment (With 6.3.7, the value set for the slot is not used)

**Comment by ndelangen** [comment]
This is implemented in the `next` branch recently, I think.
Which will be released as part of 7.0.

It's be great if you could verify the beta release meets your expectations.
